testsuite_1:
  scene_config: "scene_3"
  test_config: "test_1"
  robot: "cob4-2"
  planer_id:
    - "SBLkConfigDefault"
    - "ESTkConfigDefault"
    - "LBKPIECEkConfigDefault"
    - "BKPIECEkConfigDefault"
    - "KPIECEkConfigDefault"
    - "RRTkConfigDefault"
    - "RRTConnectkConfigDefault"
    - "RRTstarkConfigDefault"
    - "TRRTkConfigDefault"
    - "PRMkConfigDefault"
    - "PRMstarkConfigDefault"
  planning_method:
    - "cartesian"
  jump_threshold:
    - 2
  eef_step:
    - 0.01
